### ISSUE TITLE
the imp module is deprecated in favour of importlib

### ISSUE THREAD

**Issue by Artchyy** [YerevaNN/mimic3-benchmarks#130]
Hey, 

when I try to run the command   ` python -um mimic3models.in_hospital_mortality.main --network mimic3models/keras_models/lstm.py --dim 16 --timestep 1.0 --depth 2 --dropout 0.3 --mode train --batch_size 8 --output_dir mimic3models/in_hospital_mortality ` to train the NN. 

I get this error: `DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses" 
`
Changed imp to importlib, in hope that it solve this problem, but then an error occurs because module `'importlib' has no attribute 'load_source'`

Did anyone else had this problem and solved it?
